From 1707ffa1ceef7f8f444bdb4ad3477700d99feffb Mon Sep 17 00:00:00 2001
From: Vincent Bernat <vincent@bernat.ch>
Date: Mon, 4 Sep 2023 18:36:51 +0200
Subject: [PATCH] addons: vxlan: fix VNI filter on single VXLAN device

In af8d5db22b10909ff06032f1a2acc2f745d1f65b, the transformation of
`vnis` (list) to `vnisd` (dictionary) left a reference to `vnis` which
is now undefined. We just remove the line as the conversion from range
to ints is done earlier in the function.

Signed-off-by: Vincent Bernat <vincent@bernat.ch>
---
 ifupdown2/addons/vxlan.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/ifupdown2/addons/vxlan.py b/ifupdown2/addons/vxlan.py
index 084aec98..3bde50aa 100644
--- a/ifupdown2/addons/vxlan.py
+++ b/ifupdown2/addons/vxlan.py
@@ -1049,7 +1049,6 @@ def single_vxlan_device_vni_filter(self, ifaceobj, vxlan_mcast_grp):
                 self.logger.error("%s: %s (%s)" %(ifaceobj.name, vlan_vni_map, str(e)))
                 return
 
-        vnis_int = utils.ranges_to_ints(vnis)
         self.iproute2.bridge_link_update_vni_filter(ifaceobj.name, vnisd)
 
     def check_and_raise_svd_tvd_errors(self, ifaceobj):
